components/mdns/Kconfig

```diff
@@ -56,10 +56,10 @@ menu "mDNS"
     endchoice
 
     config MDNS_TASK_AFFINITY
-        hex
+        int
+        default FREERTOS_CORE0_AFFINITY if MDNS_TASK_AFFINITY_CPU0
```

Comment:
Are you going backport these macros to IDF stable releases? Currently we use the same `mdns` branch for all IDF releases `>= v5.0`

Reply:
@david-cermak There changes are going to be applied starting from ESP-IDF v5.2, so I think v5.2 will need to increase the minimum required version for mdns. Will this be an issue?

Reply:
It would be great if we can avoid it, as the `mdns` is in maintenance mode (we don't plan new features), so bugfixing would result in double work.

We usually add `#ifdefs` to the code, but I don't think it's too easy with `Kconfig` (something like?)
```
    if !FREERTOS_CORE0_AFFINITY
        config FREERTOS_CORE0_AFFINITY
            int
            default 0
    endif
```